modules/ROOT/pages/delete-identifiers.adoc

```diff
@@ -1,22 +1,28 @@
 = Delete Identifiers
 
-You might want to delete an identifier when it is no longer needed. 
+When an identifier is no longer needed, consider deleting the identifier.
 
-You can delete identifiers such as X12 ISA, X12 GS, AS2, and DUNS. You cannot delete the last identifier, as there must be at least one identifier in the list of identifiers.
+You can delete identifiers such as X12 ISA, X12 GS, AS2, and DUNS.
 
-Before you delete an identifier, consider the following: 
+Before you delete an identifier, consider the following:
```

Comment:
Consider moving any or all of these into the intro text itself. Personally, I kind of like the idea of an intro something like "Although you must always have at least one identifier per message flow, you can delete identifiers that are no longer used in that message flow [and that you no longer need? Is there a difference?] so that <reason for deleting>"

Then I would go to the steps...because the title here implies that this text is the task, rather than "Deleting."

Then I would mention the message routing thing...or maybe even include that before the steps. 

I'm not sure you need the final bullet, but maybe end with that warning and an explanation?

Another option is to consider prerequisites before Step 1.

Reply:
oops...I just can't control my CU impulses! Back to CE....

Reply:
Adjusted slightly - left list, added Steps heading, and reworded intro. PM wants the info in the way it was presented.